features/step_definitions/generic_steps.rb

```diff
@@ -55,6 +55,10 @@
   visit current_url
 end
 
+Then /^I should see "([^"]*)" before "([^"]*)"$/ do |first, second|
```

Comment:
What's the difference between this step and the `"X" should appear before "Y"` step?
https://github.com/otwcode/otwarchive/blob/d7d5b627216521741bf3af996662208f01002812/features/step_definitions/web_steps.rb#L228-L230